util/importers/ldc93s1.py

```diff
@@ -95,15 +95,17 @@ def total_batches(self):
         return int(ceil(float(len(self._txt_files)) /float(self._batch_size)))
 
 
-def read_data_sets(data_dir, batch_size, numcep, numcontext, thread_count=1, limit_dev=0, limit_test=0, limit_train=0):
+def read_data_sets(data_dir, train_batch_size, dev_batch_size, test_batch_size, numcep, numcontext, thread_count=1, limit_dev=0, limit_test=0, limit_train=0):
     # Conditionally download data
     LDC93S1_BASE = "LDC93S1"
     LDC93S1_BASE_URL = "https://catalog.ldc.upenn.edu/desc/addenda/"
     local_file = base.maybe_download(LDC93S1_BASE + ".wav", data_dir, LDC93S1_BASE_URL + LDC93S1_BASE + ".wav")
     _ = base.maybe_download(LDC93S1_BASE + ".txt", data_dir, LDC93S1_BASE_URL + LDC93S1_BASE + ".txt")
 
     # Create all DataSets, we do not really need separation
-    train = dev = test = _read_data_set(data_dir, thread_count, batch_size, numcep, numcontext)
+    train = _read_data_set(data_dir, thread_count, train_batch_size, numcep, numcontext)
```

Comment:
During refactoring for the last big PR, I thought about splitting the ds_importer_module.read_data_sets into separate calls - one per set. It could save us quite some redundancy. Now we are reading all sets for all three different set executions train/dev/test. I don't now what the exact overhead is, but at least the code would get a bit simpler.

Here we could somehow make a public version of `_read_data_set`